components/script/dom/htmlformelement.rs

```diff
@@ -465,6 +466,24 @@ impl HTMLFormElement {
     /// https://html.spec.whatwg.org/multipage/#statically-validate-the-constraints
     fn static_validation(&self) -> Result<(), Vec<FormSubmittableElement>> {
         let node = self.upcast::<Node>();
+        for child in node.traverse_preorder() {
+            if let Some(el) = child.downcast::<Element>() {
+                if el.disabled_state() {
+                    continue;
+                }
+                let validatable = match el.as_maybe_validatable() {
```

Comment:
as_maybe_validatable method is added and 
![as_maybe_validatable](https://cloud.githubusercontent.com/assets/21151022/20112834/dd9076f4-a5bb-11e6-8ce8-20190987cea7.PNG)
defined in element.rs file and hence is called as el.as_maybe_validatable.